swig/python2/Makefile.am

```diff
@@ -3,14 +3,15 @@ AM_CFLAGS = -fPIC
 PYLIBVER ?= python$(PYTHON_VERSION)
 
 AM_CPPFLAGS =	@xml2_CFLAGS@ \
+		-I${top_srcdir}/src/common/public \
 		-I$(top_srcdir)/src/CPE/public \
+		-I$(top_srcdir)/src/CVE/public \
 		-I$(top_srcdir)/src/CVSS/public \
+		-I$(top_srcdir)/src/DS/public \
 		-I$(top_srcdir)/src/OVAL/public \
+		-I${top_srcdir}/src/source/public \
 		-I$(top_srcdir)/src/XCCDF/public \
 		-I$(top_srcdir)/src/XCCDF_POLICY/public \
-		-I${top_srcdir}/src/common/public \
-		-I${top_srcdir}/src/source/public \
-		-I$(top_srcdir)/src/CVE/public \
```

Comment:
Total fan of alphabetically sorting things, if we can keep consistency between branches. Just did alphabetical sorting of rule selectors in an SSG profile (https://github.com/OpenSCAP/scap-security-guide/pull/1963)

/me ducks and runs